starter-code/index.js

```diff
@@ -1,11 +1,46 @@
 class SortedList {
-    constructor() {}
-    add(item) {}
-    get(pos) {}
-    max() {}
-    min() {}
-    average() {}
-    sum() {}
+
+    constructor() {
+      this.items=[];
+      this.length=0;
+    }
+
+    add(item) {
+      this.items.push(item);
+      this.length +=1;
+    }
+
+    get(pos) {
```

Comment:
Il faut que tu throw les erreurs : 
  get(pos) {
    if (pos > this.length) {
      throw new Error("OutOfBounds");
    }
    return this.items[pos];
  }